centreon/src/Core/ServiceCategory/Infrastructure/Configuration/symfony.yaml

```
@@ -0,0 +1,25 @@
+
+    Core\ServiceCategory\Application\Repository\ReadServiceCategoryRepositoryInterface:
+      class: Core\ServiceCategory\Infrastructure\Repository\DbReadServiceCategoryRepository
+
+    Core\ServiceCategory\Infrastructure\Repository\DbWriteServiceCategoryActionLogRepository:
+        decorates: Core\ServiceCategory\Application\Repository\WriteServiceCategoryRepositoryInterface
+        arguments:
+        - '@.inner'
+        - '@Core\ActionLog\Application\Repository\WriteActionLogRepositoryInterface'
+        - '@Core\ServiceCategory\Application\Repository\ReadServiceCategoryRepositoryInterface'
+        - '@Centreon\Domain\Contact\Interfaces\ContactInterface'
+        - '@Centreon\Infrastructure\DatabaseConnection'
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)